building-a-preview/index.md

```diff
@@ -0,0 +1,777 @@
+
+In addition to speeding up Preview builds, Base Previews can help you
+dramatically reduce the size of Previews built from that Base Preview. This is
+because the Base Preview contains everything Tugboat needs to run your Preview,
+while subsequent Previews only contain the differences between the Base Preview
+and the new Preview build.
+
+In practice, this means that a Base Preview might be 3GB in size, but subsequent
+Previews might be only 100MB.
+
+Ready to set up a Base Preview? Check out:
+[Set a Base Preview](#set-a-base-preview).
+
+#### Use dummy data
+
+Are you pulling in a large production database? You can save Preview space -
+_and_ speed up your Preview builds at the same time - by switching to a small
+dummy database that contains enough data for testing, but doesn't mirror your
+large production behemoth.
+
+### Contact Tugboat support for help optimizing your Config file
+
+Sometimes, speeding up your Preview builds can be as simple as having a second
+set of eyes take a look at your Config file and make recommendations to help you
+optimize it. This might include something like running a command in `init`
+instead of `update`, or scripting a few simple commands. The team at Tugboat is
+happy to help; our [Support](../support/index.md) page can direct you to our
+Tugboat support Slack, or an email address where you can reach us.
+
+### Upgrade your project tier to a higher-performance tier
+
+If it's not a simple question of optimizing your Config file - for example, if
+you're building a complex or resource-intensive sequence of code in every
+Tugboat Preview - you might want to consider upgrading your Tugboat Project to a
+higher-performance tier. A higher-performance tier gives you more CPU power and
+RAM to build your Previews, which can mildly or dramatically speed up your build
+times. When build times matter, keep this option in mind.
+
+You can change your Tugboat plan in **Project Settings**.
+
+## Preview size explained
+
+What Tugboat calls "Preview size" is actually the size of the container in which
+the Preview lives. When a Preview is done building, Tugboat takes a snapshot of
+the container at that moment in time, and that's what you're seeing in your
+Tugboat Dashboard.
+
+When you [set up Services](../setting-up-services/index.md) in your Preview,
+Tugboat pulls those
+[Service images](../setting-up-services/index.md#specify-a-service-image) into
+the Preview container. Each of these Service images contributes to the total
+size of the Preview when it is fully built.
+
+For example, say I'm building a Preview that uses Tugboat's Service images for
+`apache`, `mysql` and `redis`; those Service images are 154MB, 226MB, and 147MB
+at the time of this writing. That's over 500MB for just these Services; by the
+time you add a database file, the container's operating system and other assets,
+you'll likely be looking at 800MB to 1GB for a Preview that's only pulling in
+100MB of code from your linked git repo.
```

Comment:
100MB is a *lot* of code. Let's change this to 100KB which may be a bit on the small side, but really drives home the point that your code is just a small part of the total disk space used by a preview.